models/link.js

```diff
@@ -21,13 +21,29 @@ export default sequelize.define('links', {
       },
     },
   },
+  goDescription: {
+    type: DataTypes.STRING,
+    allowNull: false,
+    validate: {
+      notEmpty: true
+    }
+  },
+  publicGO: {
+    type: DataTypes.STRING,
+    allowNull: false,
+    validate: {
+      notEmpty: true
+    },
+  },
 }, {
   scopes: {
```

Comment:
Look at how we do scopes in other models. You should add one here for filtering links that are public.